src/scripts/events/ethereum-events-import.ts

```diff
@@ -0,0 +1,104 @@
+import { CommunityConference } from "@/lib/types"
+
+import "dotenv/config"
+
+async function getPageMetadata(url: string): Promise<Record<string, string>> {
+  try {
+    const metaTags: Record<string, string> = {}
+    const text = await fetch(url).then((r) => r.text())
+
+    const description = text.match(
+      /<meta.*?name="description".*?content="(.*?)".*?>|<meta.*?content="(.*?)".*?name="description".*?>/i
+    )
+    if (description && Array.from(description).length > 2)
+      metaTags["description"] = Array.from(description)[1]
+
+    const tags = text.matchAll(
+      /<meta (property|name)="(og|twitter):(\S+)" content="(\S+)"/gm
+    )
+    for (const matchGroup of Array.from(tags)) {
+      const key = matchGroup[3]
+      const value = matchGroup[4]
+      if (key && value) metaTags[key] = value
+    }
+
+    return metaTags
+  } catch (error) {
+    console.error("Unable to fetch metadata", url)
+    return {}
+  }
+}
+
+export async function EthereumEventsImport() {
+  const googleApiKey = process.env.GOOGLE_API_KEY
+  const sheetId = "1NEu_FCc1hnGAuRgPmbXXpf0h2lCrCOlMKbbFEqgkVDQ"
+
+  if (!googleApiKey || !sheetId) {
+    console.warn("googleApiKey or sheetId not set")
```

Comment:
We're hard-coding `sheetId`, thus will always be available:

```suggestion
  if (!googleApiKey) {
    console.warn("googleApiKey not set")
```

Reply:
Good catch!